app/models/template.rb

```diff
@@ -1,7 +1,8 @@
 class Template < ActiveRecord::Base
   validates_lengths_from_database
 
-  attr_accessible :name, :template, :snippet, :audit_comment, :location_ids, :organization_ids, :locked, :default
```

Comment:
I don't know if it's intentional, but I think it's broadly correct to prevent mass-assignment of locked through the API/UI etc, since it's not meant to be changed by users in that way.  We have a model-level auth check (Template#template_changes) so although the attribute's protected, this is probably the right direction.

In foreman_bootdisk I changed mass-assignment to simple assignment: https://github.com/theforeman/foreman_bootdisk/commit/793648ecfcd87f75d4f953914bfd2b03ee346f33